pyccel/stdlib/external/fitpack.py

```diff
@@ -1,4 +1,9 @@
+#--------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See file LICENSE or #
+# go to https://github.com/pyccel/pyccel/blob/master/LICENSE for full license details. #
+#--------------------------------------------------------------------------------------#
 #$ header metavar print=True
+# pylint: disable=missing-module-docstring
```

Comment:
switch lines 5 and 6, this line `#$ header metavar print=True`, it's a part of code, not a comment.